pkg/utils/utils.go

```diff
@@ -26,15 +26,22 @@ import (
 )
 
 func RandDate(rnd *rand.Rand) string {
-	return RandTime(rnd).Format("2006-01-02")
+	min := time.Date(1970, 0, 0, 0, 0, 0, 0, time.UTC).Unix()
+	max := time.Date(2024, 1, 0, 0, 0, 0, 0, time.UTC).Unix()
+
+	sec := rnd.Int63n(max-min) + min
+	return time.Unix(sec, 0).UTC().Format("2006-01-02")
```

Comment:
why `RandTime(rnd).Format("2006-01-02")` did not work for you ?

Reply:
Redid this function.